clients/banking/src/pages/AccountDetailsIbanPage.tsx

```diff
@@ -23,7 +23,8 @@ import { ErrorView } from "../components/ErrorView";
 import { LakeCopyTextLine } from "../components/LakeCopyTextLine";
 import { AccountDetailsIbanPageDocument } from "../graphql/partner";
 import { formatNestedMessage, t } from "../utils/i18n";
-import { printIbanFormat } from "../utils/iban";
+
```

Comment:
Remove the extra newline (prettier-organize-imports organize them by "group"), adding a new line mess with this logic.